README.md

```diff
@@ -302,18 +310,51 @@ function MySection {
 ```
 
 ## 🖊️ Styling Components
-This starter uses [Tailwind CSS](https://tailwindcss.com/) for styling. If you want, you can remove it and use other solutions. Both Gatsby and Store UI support many different CSS frameworks, like [emotion](https://emotion.sh/docs/introduction), [stitches](https://stitches.dev/docs/introduction) or even no CSS framework at all. Check the supported frameworks at [the Gatsby website](https://www.gatsbyjs.com/docs/how-to/styling/built-in-css/#other-css-options)
-This guide covers best practices and patterns to use when styling with Tailwind.
 
-Tailwind is a utility-first CSS framework. The goal of this starter is to make it possible for developers to use Tailwind themes to change the look of the store. To accomplish this, a few things need to be respected:
+Our customized theme is based on [Design Tokens](https://css-tricks.com/what-are-design-tokens/) using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) of a `class` for each one of them. Today, we have the following files in the `src/styles` folder:
```

Comment:
I couldn't understand the first sentence very well, particularly the part "of a `class` for each one of them". When you say "class", is that a CSS class selector, or maybe JS's `class` construct?
:\

Reply:
```suggestion
Our customized theme is based on [Design Tokens](https://css-tricks.com/what-are-design-tokens/) using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) or a `css class` for each one of them. Today, we have the following files in the `src/styles` folder:
```

Reply:
I like the following formatting for "CSS class"  better:
```suggestion
Our customized theme is based on [Design Tokens](https://css-tricks.com/what-are-design-tokens/) using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) or a CSS class for each token. Today, we have the following files in the `src/styles` folder:
```